util/patricia_trie/src/fatdb.rs

```diff
@@ -70,18 +70,18 @@ where
 
 /// Itarator over inserted pairs of key values.
 pub struct FatDBIterator<'db, H, C>
-where 
-	H: Hasher + 'db, 
-	C: NodeCodec<H> + 'db
+where
+	H: Hasher + 'db,
+	C: NodeCodec<H::Out> + 'db
```

Comment:
in next pr, it would be good to remove these constrains from struct declaration, cause they are redundant